### ISSUE TITLE
App Manager app won't turn on 

### ISSUE THREAD

**Issue by reyis4402** [MuntashirAkon/AppManager#563]
I have updated the application to the latest version The application began not to open. 

Devives
ArrowOS Android 11

**Comment by reyis4402** [comment]
The application remains on the continuous opening screen

**Comment by reyis4402** [comment]
https://user-images.githubusercontent.com/60405617/130345900-b7e87589-955e-4108-ac0a-b41da6192f0d.mp4

**Comment by MuntashirAkon** [comment]
Did you restart your device after updating?

**Comment by reyis4402** [comment]
Yes, but the problem continues

**Comment by MuntashirAkon** [comment]
It's difficult to understand the problem without logs. You can sent logs via email if you have privacy issues.

**Comment by reyis4402** [comment]
[2021-08-22-17-03-32.txt](https://github.com/MuntashirAkon/AppManager/files/7027496/2021-08-22-17-03-32.txt)

**Comment by MuntashirAkon** [comment]
> [2021-08-22-17-03-32.txt](https://github.com/MuntashirAkon/AppManager/files/7027496/2021-08-22-17-03-32.txt)

Since your OS is highly modified, it's difficult to tell what went wrong. Maybe it's a problem with AM, or LSPosed or XPrivacyLua is causing issues. So, take the logs again but use [a debug build](https://github.com/MuntashirAkon/AppManager/actions/runs/1133078108) this time.

**Comment by fusionneur** [comment]
same happens for me, but eventually opens. It takes minutes to open since latest updates..

**Comment by MuntashirAkon** [comment]
> update: i don't know what happenned but i rebooted several times these days and now it opens instantly.. might have been related to latest lsposed version that got updated recently..

Yeah, it could be. AM calls a lot of private APIs which you'd never see in an app that doesn't depend on Xposed/LSPosed/Zygote-patches. Since they heavily control what other apps can access, it can have an impact on AM like in the logs above.

**Comment by MuntashirAkon** [comment]
Closed due to lack of response. Feel free to open it again if you're still facing the issue.

**Comment by fusionneur** [comment]
it worked fine recently. thanks!
